inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/node/ExtractNodeProviderFactory.java

```diff
@@ -19,30 +19,25 @@
 
 import org.apache.inlong.manager.common.enums.ErrorCodeEnum;
 import org.apache.inlong.manager.common.exceptions.BusinessException;
-import org.apache.inlong.manager.pojo.sort.node.extract.HudiProvider;
-import org.apache.inlong.manager.pojo.sort.node.extract.KafkaProvider;
-import org.apache.inlong.manager.pojo.sort.node.extract.MongoProvider;
-import org.apache.inlong.manager.pojo.sort.node.extract.MysqlBinlogProvider;
-import org.apache.inlong.manager.pojo.sort.node.extract.OracleProvider;
-import org.apache.inlong.manager.pojo.sort.node.extract.PostgreSqlProvider;
-import org.apache.inlong.manager.pojo.sort.node.extract.PulsarProvider;
-import org.apache.inlong.manager.pojo.sort.node.extract.RedisProvider;
-import org.apache.inlong.manager.pojo.sort.node.extract.SqlServerProvider;
-import org.apache.inlong.manager.pojo.sort.node.extract.TubeMqProvider;
-import org.apache.inlong.manager.pojo.source.StreamSource;
-import org.apache.inlong.sort.protocol.node.ExtractNode;
-
-import com.google.common.collect.Lists;
-import org.apache.commons.collections.CollectionUtils;
+import org.apache.inlong.manager.pojo.sort.node.base.ExtractNodeProvider;
+import org.apache.inlong.manager.pojo.sort.node.provider.HudiProvider;
+import org.apache.inlong.manager.pojo.sort.node.provider.KafkaProvider;
+import org.apache.inlong.manager.pojo.sort.node.provider.MongoProvider;
+import org.apache.inlong.manager.pojo.sort.node.provider.MySQLBinlogProvider;
+import org.apache.inlong.manager.pojo.sort.node.provider.OracleProvider;
+import org.apache.inlong.manager.pojo.sort.node.provider.PostgreSQLProvider;
+import org.apache.inlong.manager.pojo.sort.node.provider.PulsarProvider;
+import org.apache.inlong.manager.pojo.sort.node.provider.RedisProvider;
+import org.apache.inlong.manager.pojo.sort.node.provider.SQLServerProvider;
+import org.apache.inlong.manager.pojo.sort.node.provider.TubeMqProvider;
 
 import java.util.ArrayList;
 import java.util.List;
-import java.util.stream.Collectors;
 
 /**
- * The node provider factory.
+ * Factory of the extract node provider.
  */
-public class NodeProviderFactory {
+public class ExtractNodeProviderFactory {
```

Comment:
Maybe should put this class from the `pojo` into the `service` directory.

Reply:
> Maybe should put this class from the `pojo` into the `service` directory.


The structure can be optimized in the next step, and the current changes should not be too large.